README.asciidoc

```diff
@@ -44,10 +44,10 @@ Kakoune has a strong focus on interactivity, most commands provide immediate
 and incremental results, while still being competitive (as in keystroke count)
 with Vim.
 
-Kakoune works on selections, which are oriented, inclusive range of characters,
-selections have an anchor and a cursor character. Most commands move both of
-them, except when extending selection where the anchor character stays fixed
-and the cursor one moves around.
+Kakoune works on selections, which are a direction-oriented, inclusive range of characters
```

Comment:
This confuses me, the original meaning is that selections are oriented (first, last) != (last, first), and inclusive (first is an element of (first, last), last as well).